Mathlib/Analysis/Normed/Order/UpperLower.lean

```diff
@@ -154,3 +240,64 @@ theorem IsLowerSet.exists_subset_ball (hs : IsLowerSet s) (hx : x ∈ closure s)
 #align is_lower_set.exists_subset_ball IsLowerSet.exists_subset_ball
 
 end Fintype
+
+section Finite
+
```

Comment:
`is_antichain.interior_eq_empty` seems to be missing from the forward-port here; if this is deliberate, there should be a `noalign`